ui/src/views/Settings.vue

```diff
@@ -22,16 +22,283 @@
-              :label="$t('settings.test_field')"
-              v-model="testField"
-              :placeholder="$t('settings.test_field')"
+          <cv-skeleton-text
+            v-if="loading.getConfiguration || loading.getDefaults"
+            heading
+            paragraph
+            :line-count="15"
+            width="80%"
+          ></cv-skeleton-text>
+          <cv-form v-else @submit.prevent="configureModule">
+            <NsTextInput
+              :label="$t('settings.hostname')"
+              :placeholder="$t('settings.hostname_placeholder')"
+              v-model="hostname"
+              class="mg-bottom"
+              :invalid-message="error.hostname"
               :disabled="loading.getConfiguration || loading.configureModule"
-              :invalid-message="error.testField"
-              ref="testField"
-            ></cv-text-input>
+              ref="hostname"
+              tooltipAlignment="center"
+              tooltipDirection="right"
+            >
+              <template slot="tooltip">
+                <div
+                  v-html="
+                    $t(
+                      'settings.hostname_must_be_relevant_for_user_authentication'
+                    )
+                  "
+                ></div>
+              </template>
+            </NsTextInput>
+            <NsComboBox
+              v-model.trim="ldap_domain"
+              :autoFilter="true"
+              :autoHighlight="true"
+              :title="$t('settings.ldap_domain')"
+              :label="$t('settings.choose_ldap_domain')"
+              :options="domains_list"
+              :acceptUserInput="false"
+              :showItemType="true"
+              :invalid-message="$t(error.ldap_domain)"
+              :disabled="loading.getConfiguration || loading.configureModule"
+              tooltipAlignment="start"
+              tooltipDirection="top"
+              ref="ldap_domain"
+            >
+              <template slot="tooltip">
+                {{ $t("settings.choose_the_ldap_domain_to_use") }}
+              </template>
+            </NsComboBox>
+            <cv-toggle
+              value="letsEncrypt"
+              :label="$t('settings.request_https_certificate')"
+              v-model="isLetsEncryptEnabled"
+              :disabled="loading.getConfiguration || loading.configureModule"
+              class="mg-bottom"
+            >
+              <template slot="text-left">{{
+                $t("settings.disabled")
+              }}</template>
+              <template slot="text-right">{{
+                $t("settings.enabled")
+              }}</template>
+            </cv-toggle>
+            <!-- advanced options -->
+            <cv-accordion ref="accordion" class="maxwidth mg-bottom">
+              <cv-accordion-item :open="toggleAccordion[0]">
+                <template slot="title">{{ $t("settings.advanced") }}</template>
+                <template slot="content">
+                  <cv-toggle
```

Comment:
I would use `NsToggle` (it supports tooltips) and show the tooltip next to toggle label (see also the comment about Web Admin link)

Reply:
done